src/aegis_ai/osidb_bot/bot.py

```
@@ -488,9 +493,11 @@ async def process_cve(self, cve: CVEID) -> None:
                 self.pending[flaw_updater.position()] = True  # mark as pending
             await flaw_updater.do()
 
-        except RuntimeError as e:
+        except BaseException as e:
             # something has failed
-            logger.warning(f"{cve}: {str(e)}")
+            handled_failure = isinstance(e, RuntimeError)
+            if handled_failure:
+                logger.warning(f"{cve}: {str(e)}")
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)